r-package-manager-demo/README.md

```diff
@@ -0,0 +1,136 @@
+# Package Manager Demo
+
+[![package-manager-demo](https://github.com/rstudio/package-manager-demo/actions/workflows/package-manager-demo.yml/badge.svg)](https://github.com/rstudio/package-manager-demo/actions/workflows/package-manager-demo.yml)
```

Comment:
Can move the badges to the top level

Reply:
If we have the badges here, then they show up on the README when you load it on RSPM (e.g, look a the readme on solo). Should we leave them here and also have them on the top-level readme?